index.js

```diff
@@ -1,5 +1,3 @@
-// @ts-check
```

Comment:
Why remove these? This forces TS checks in JS files, can we keep it in? Or is there another way you enabled the checks that I missed?

Reply:
added them back and made deno run the tests without type checking.